app/section/sections/mobile-operating-systems/manifest.json

```diff
@@ -0,0 +1,30 @@
+{
+  "id": "mobile-operating-systems",
+  "title": "Native mobile or tablet - supported operating systems",
+  "mainAdvice": "Answer the following questions about your native mobile or tablet Solution",
+  "additionalAdvice": [
+    "This information is displayed on the Marketing page for buyers to view."
+  ],
+  "questions": {
+    "operating-systems": {
+      "mainAdvice": "What devices or operating systems are supported? Select all that apply",
+      "additionalAdvice": "Provide the details you would like to be point of contact for this Solution.",
```

Comment:
Review additional advice. Think this is from the contact details section

Reply:
haha oops I think I changed the contact details one (didn't commit don't worry)